### PR TITLE
add shop banniere

### PR DESCRIPTION
**Pull request by RomainLF**

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok

File: frontend/src/components/BanShop/style.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      font-size: 3.5vmax;
+      line-height: 150%;
+    }
+    h2 {
+      font-size: 1.9vmax;
+      margin-top: 2%;
+    }
+  }
+  .mainImages {
+    //background-color: red;
```
Comment (LuigiMenez) [inline frontend/src/components/BanShop/style.jsx:35]: Dead Code

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Dead code

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**